1.Data Collection.ipynb

```diff
@@ -0,0 +1,640 @@
+{
```

Comment:
let's remove 1. and 2. from file names, because it violates the convention. 

Use names
`Data_Collection.ipynb`
`Resource_Recommendation_Algorithm.ipynb` (this recommend resources right?)

Reply:
Done.